numpy/lib/function_base.py

```diff
@@ -310,10 +310,10 @@ def average(a, axis=None, weights=None, returned=False):
         of the weights as the second element. The return type is `Float`
         if `a` is of integer type, otherwise it is of the same type as `a`.
```

Comment:
> The return type is `Float` if `a` is of integer type, otherwise it is of the same type as `a`.

Please remove that sentence. It's the one that's causing trouble.